frigate/config.py

```diff
@@ -463,6 +463,9 @@ class FfmpegConfig(FrigateBaseModel):
         default_factory=FfmpegOutputArgsConfig,
         title="FFmpeg output arguments per role.",
     )
+    timeout: float = Field(
```

Comment:
I don't think timeout is a good name as it's confused with the actual meaning of timeout (how long of no communication should result in an error).

I think `healthcheck_interval` might be better